test/config.test.ts

```diff
@@ -324,6 +324,14 @@ describe("config", () => {
         })
     );
 
+    it(
+        "arrow-function-parameters",
+        assertSchema("arrow-function-parameters", {
+            type: "Parameters<typeof myFunction>",
```

Comment:
Let's just test MyType and define it to be `Parameters<typeof myFunction>` in the code. Separate out the selector test into a separate test case.

Reply:
I'm not entirely sure I follow, you want me to define a type `type MyType = Parameters<typeof myFunction>` in the test fixture? I'm trying to introduce something to the lib so that the user does not have to define that type explicitly, so with the ambitions of the PR I think the current test case makes sense

Reply:
The way I see it (and I may be missing something), this pull request introduces two changes. First, it adds support for Parameters (and maybe ReturnType). Second, it adds support for specifying more than a type name to the cli or the `type` param. I see those as separate features that should be in two prs. I hope it doesn't make too much trouble. 

If I miss something, I'm happy to have a single pull request instead.

Reply:
This only adds one change, and that  parsing of the function parameters and including that in the outputted JSON schema. Semantically you could say that we're introducing two changes since now it will also support `Parameters<typeof myFunction>` as a type name, but I don't think I can split this out as it currently stands into multiple PRs. 

In regards to adding support of `Parameters` & `ReturnType` as type utility functions I'd be happy to add support for it in other separate PR's (after we've resolved this PR :) )

Reply:
Sounds good. Let's do that.